<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ThermoLogger v1</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>USB Communication</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="com_port_usb"/>
           </item>
           <item>
            <widget class="QPushButton" name="com_port_bt">
             <property name="text">
              <string>Connect!</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="com_port_info">
             <property name="text">
              <string>Info</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>ThermoCouple 1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="tc1_status">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_8">
               <property name="title">
                <string>Internal T</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_36">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_35">
                  <item>
                   <widget class="QLabel" name="tc1_int_temperature">
                    <property name="minimumSize">
                     <size>
                      <width>38</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Box</enum>
                    </property>
                    <property name="text">
                     <string>0.0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_19">
                    <property name="maximumSize">
                     <size>
                      <width>31</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="text">
                     <string>[ºC]</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_9">
               <property name="title">
                <string>External T</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_37">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_38">
                  <item>
                   <widget class="QLabel" name="tc1_ext_temperature">
                    <property name="minimumSize">
                     <size>
                      <width>38</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Box</enum>
                    </property>
                    <property name="text">
                     <string>0.0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_20">
                    <property name="maximumSize">
                     <size>
                      <width>31</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="text">
                     <string>[ºC]</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="tc1_graph">
               <property name="text">
                <string>Plot Internal T1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="tc2_graph">
               <property name="text">
                <string>Plot External T1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>ThermoCouple 2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Status:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="tc2_status">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_10">
               <property name="title">
                <string>Internal T</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_39">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_40">
                  <item>
                   <widget class="QLabel" name="tc2_int_temperature">
                    <property name="minimumSize">
                     <size>
                      <width>38</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Box</enum>
                    </property>
                    <property name="text">
                     <string>0.0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_21">
                    <property name="maximumSize">
                     <size>
                      <width>31</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="text">
                     <string>[ºC]</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="horizontalGroupBox_11">
               <property name="title">
                <string>External T</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_41">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_42">
                  <item>
                   <widget class="QLabel" name="tc2_ext_temperature">
                    <property name="minimumSize">
                     <size>
                      <width>38</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Box</enum>
                    </property>
                    <property name="text">
                     <string>0.0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_22">
                    <property name="maximumSize">
                     <size>
                      <width>31</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font">
                     <font>
                      <family>Consolas</family>
                      <pointsize>10</pointsize>
                      <stylestrategy>PreferAntialias</stylestrategy>
                     </font>
                    </property>
                    <property name="text">
                     <string>[ºC]</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="tc1_graph_2">
               <property name="text">
                <string>Plot Internal T2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="tc1_graph_3">
               <property name="text">
                <string>Plot External T2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="horizontalGroupBox_12">
          <property name="title">
           <string>Sampling rate</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_43">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_44">
             <item>
              <widget class="QComboBox" name="tc1_sampling_rate">
               <item>
                <property name="text">
                 <string>150ms</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>250ms</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>500ms</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1s</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5s</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10s</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15 min</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="horizontalGroupBox_13">
          <property name="title">
           <string>Graph Timeline</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_45">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_46">
             <item>
              <widget class="QComboBox" name="tc1_sampling_rate_2">
               <item>
                <property name="text">
                 <string>1 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30 min</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1 hour</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5 hour</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Complete</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="graph_start">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_81">
        <item>
         <widget class="QLabel" name="record_data_label">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
            <stylestrategy>PreferAntialias</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_recording_data">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Record Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_recording_data_mark">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Mark</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="dl_debug">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
